lib/src/__tests__/DateTimePicker.test.tsx

```diff
@@ -9,6 +9,23 @@ import { DateTimePicker, DateTimePickerProps } from '../DateTimePicker/DateTimeP
 
 const format = process.env.UTILS === 'moment' ? 'MM/DD/YYYY HH:mm' : 'MM/dd/yyyy hh:mm';
 
+describe('DateTimePicker', () => {
+  describe('prop: mask', () => {
+    it('should handle the mark prop', () => {
+      const component = mount(
+        <DateTimePicker
+          renderInput={props => <TextField {...props} />}
+          label="DateTimePicker"
+          onChange={() => {}}
+          mask="__.__.____ _:__ _M"
+          value={utilsToUse.date('2018-01-01T00:00:00.000Z')}
+        />
+      );
+      expect(component.find('input').prop('value')).toBe('01.01.2018 1:00 AM');
```

Comment:
This won't work unless you also somehow change the locale used in the utils. In fact, I'm not sure if the mask has anything to do with how the component maps Date values to strings for the TextField.

Perhaps you could change to `value={null}` above and change the assertion to output the correct mask?